app/controllers/application_controller.rb

```diff
@@ -2,4 +2,9 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :configure_permitted_parameters, if: :devise_controller?
+      protected
```

Comment:
Indent access modifiers like `protected`.<br>Keep a blank line before and after `protected`.